modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,98 @@
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * post_installation_configuration/post-install-bare-metal-configuration.adoc
+
+ifeval::["{context}" == "ipi-install-post-installation-configuration"]
+:postinstall-bare-metal-ipi:
+endif::[]
+ifeval::["{context}" == "post-install-bare-metal-configuration"]
+:postinstall-bare-metal-upi:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-manifest-file-multiple-interfaces-br-ex_{context}"]
+= Optional: Creating an additional OVN-Kubernetes `br-ex` bridge network
+
+You can create an additional `br-ex` bridge network that has a different configuration to the default `br-ex` bridge. Creating an additional `br-ex` bridge requires a change to the `configure-ovs.sh` shell script. Changing the shell script requires that you create a machine config manifest file for the network plugin of the Secure Neutron Networking implementation
+
+After you create and edit the manifest file, the Machine Config Operator completes tasks in the following order:
+
+* Runs the `configure-ovs.sh` shell script.
+* Drains all nodes in your cluster.
+* Inject Ignition configuration files into each node in your cluster, so that each node receives the additional `br-ex` bridge network configuration.
+* Uncordons the nodes. After all the nodes return to the `Ready` state, the Operator applies the `k8s.ovn.org/l3-gateway-config` annotation to each node.
+
+For more information about useful situations for the additional `br-ex` bridge and a situation that always requires the default `br-ex` bridge, see "Configuration for a localnet topology"
+
+.Prerequisites IF ANY?
+
+
+.Procedure
+
+. Create an Open vSwitch (OVS) bridge in the `etc/ovnk/extra_bridge` directory of your OVN-Kubernetes network plugin:
++
+.Example configuration for the `extra_bridge` file
+[source,text]
+----
+DEVICE=eno1 <1>
+TYPE=OVSPort
+DEVICETYPE=ovs
+OVS_BRIDGE=<additional_br-ex_bridge_name> <2>
+ONBOOT=yes
+NM_CONTROLLED=no
+BOOTPROTO=none
+----
+<1> This field supports `bond1` or `eno1` bond interface types.
+<2> Where `<additional_br-ex_bridge_name>` is the name of your additional `br-ex` bridge.
+
+. Prepare the environment variables. Replace `<machine_role>` with the node role, such as `worker`, and replace `<interface_name>` with the name of your additional `br-ex` bridge name.
++
+[source,terminal]
+----
+$ export ROLE=<machine_role>
+----
++
+[source,terminal]
+----
+$ export INTERFACE=<interface_name>
+----
+
+. Create a machine config manifest file for configuring the network plugin of the Secure Neutron Networking implementation by entering the following command in your terminal:
++
+[source,terminal]
+----
+$ oc create -f sec-bridge-cni.yaml
+----
+
+. Edit the machine config YAML file to have a configuration analogous to the following example:
```

Comment:
machine config definition is accurate - shows what I was highlighting above re: definition. Let me know if you've got any questions about this!